src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Figures {
```

Comment:
Inteface provides some behaviour, name should represent this behaviour
```suggestion
public interface Descriptive {
```